mixins/localize/localize-mixin.js

```diff
@@ -116,9 +91,13 @@ export const _LocalizeMixinBase = dedupeMixin(superclass => class LocalizeMixinC
 		const translatedMessage = new IntlMessageFormat(value, language);
 		let formattedMessage = value;
 		try {
-			if (Object.values(params).some(v => typeof v === 'function')) throw 'localize() does not support rich text.';
+			validateMarkup(formattedMessage, noAllowedTagsRegex);
 			formattedMessage = translatedMessage.format(params);
 		} catch (e) {
+			if (e.name === 'MarkupError')	{
```

Comment:
Are we worried that places might have been mistakenly doing this and getting encoded tags and will now be getting nothing?

Reply:
It's definitely possible. I'll run through the list linked above and do an audit. Most of them look fine, though, at a glance.

Reply:
Audited the uses of tags.

- [ ] [sequences](https://github.com/BrightspaceHypermediaComponents/sequences/blob/9b694b3bf2ec33881285b6c44f86dd812057b374/components/end-of-lesson/d2l-sequences-content-eol-in-progress.js#L171) - This seems like the only legitimate one that will break, and can be cleaned up significantly by using `localizeHTML`
- [ ] [d2l-class-stream](https://github.com/Brightspace/d2l-class-stream-ui/blob/8fe7e830b21de738ea21d38a48db8d21173e1c96/components/d2l-cs-activity-message-info.js#L201) - In Polymer, but would break if we switched the behavior to use this stuff
- [ ] [creator-plus-authoring-tools](https://github.com/search?q=repo%3ABrightspaceUI%2Fcreator-plus-authoring-tools%20path%3Aen.js%20%22%3C%22&type=code) - A couple terms with tags, but looks like they aren't even used. I'll reach out to confirm.
- [ ] [homepage-widget-expansion-pack](https://github.com/Brightspace/homepage-widget-expansion-pack/blob/833e00cf7e675fda921a679497baf7c03a4a96bd/lang/en.js#L40-L51) - Has some tags, but doesn't look like they're used with the mixin